### PR TITLE
New kudo jar design

### PR DESCRIPTION
**Pull request by CanolKabisa**

### CODE REVIEW

File: app/assets/javascripts/jar.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    if(loaded == max) {
+        loaded = 0;
+        stopLoading();
+        $(".dynamic").fadeIn(1000);
+        $(".loadingmessage").hide();
+
+    // setTimeout(startLoading, 1000);
+    }
+    else loaded = loaded + 1
+
```
Comment (remcohh) [inline app/assets/javascripts/jar.js]: This is what i meant:
http://www.w3schools.com/js/js_switch.asp

File: app/assets/javascripts/jar.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      $(".dynamicmessage").html("over the half full mark. Nice, but we're not there yet ");
+
+    if(loaded == 75)
+        $(".dynamicmessage").html("is not full yet. Why is this not full yet? ");
+
+    if(loaded == 90)
+        $(".dynamicmessage").html("almost full. Can it be true? Are we almost there? Soooo close, we got this! ");
+
+}
+
```
Comment (remcohh) [inline app/assets/javascripts/jar.js]: Perhaps we should user translation files for texts:
https://github.com/fnando/i18n-js

File: app/assets/javascripts/jar.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+//var message = document.getElementsByClassName('dynamicmessage');
+$( document ).ready(function() {
+    $(".dynamic").hide();
+
+});
+
+function increment() {
+    $('#counter').html(loaded+' Coins');
+    $('#drink').css('top', (100-loaded*.9)+'%');
+  //  console.log("test", loaded, max, (loaded==5 && loaded < max) );
```
Comment (ariejan) [inline app/assets/javascripts/jar.js]: Before merging, please remove commented-out code. Apparently it's not needed, so remove it.

File: app/assets/javascripts/jar.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    for (i = 0; i < 20; i++) {
+      if(loaded == i * 5 && loaded < max) $('#cubes div:nth-child(' + i +')').fadeIn(100);
+    }
+    if(loaded == max) {
```
Comment (ariejan) [inline app/assets/javascripts/jar.js]: Can `loaded >= max` happen? If so, what happens with this code?

**Comment by ariejan** [comment]
This is now based off of `epic/refactoring` as there will be breaking changes (apart from this PR) introduced that all need to be resolved before going to `develop` and `master`.

**Pull request closed without merging**